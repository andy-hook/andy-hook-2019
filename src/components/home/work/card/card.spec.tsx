import React from "react"
import renderer from "react-test-renderer"

import Card from "./card"

const props = {
  label: "Card label",
  desc: "Card description",
  href: "https://www.google.com",
}

describe("<Work />", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Card {...props} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
