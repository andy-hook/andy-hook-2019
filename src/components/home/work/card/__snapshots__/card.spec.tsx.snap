// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Work /> renders correctly 1`] = `
Array [
  "Card label",
  "Card description",
  "https://www.google.com",
]
`;
